parlai/utils/misc.py

```diff
@@ -479,6 +433,32 @@ def __exit__(self, exc_type, exc_value, exc_traceback):
         pass
 
 
+def round_sigfigs(x: Union[float, 'torch.Tensor'], sigfigs=4) -> float:
+    """
+    Round value to specified significant figures.
+
+    :param x: input number
+    :param sigfigs: number of significant figures to return
+
+    :returns: float number rounded to specified sigfigs
+    """
+    x_: float
```

Comment:
this is a thing now??

Reply:
yeah useful if you want to declare a variable that's going to be set in two different code paths. it's not entirely necessary though.